engineering/frontend/logging-monitoring.md

```diff
@@ -0,0 +1,20 @@
+# Logging and Monitoring
+
+This document in intended to let you know how we monitoring the frontend applications at Dwarves
```

Comment:
I think we should spend a large part of the document explaining which metrics should be measured and how we set up notification for those metrics. Let me try to propose a draft for it.